docs/source/system_reference_guide/jobsui.ipynb

```diff
@@ -15,11 +15,11 @@
    "id": "db3a214d-4c06-426e-a00c-1ffc8fec73b2",
    "metadata": {},
    "source": [
-    "## Access\n",
+    "## Access the Jobs UI\n",
     "\n",
-    "1. From your workspace click on the __View & Submit Jobs__ card on the launcher tab.\n",
+    "1. From your workspace click on the __View & Submit Jobs__ card on the >auncher tab.\n",
```

Comment:
```suggestion
    "1. From your workspace click on the __View & Submit Jobs__ card on the Launcher tab.\n",
```

Reply:
in https://github.com/MAAP-Project/maap-documentation/pull/347